File: languages/go/test/compat_test.go
//go:build compat

package test

import (
	"testing"

	"github.com/bloock/bloock-sdk-go/v2/client"
	"github.com/stretchr/testify/assert"
)

func TestCompat(t *testing.T) {
	recordClient := client.NewRecordClient()
	record, err := recordClient.FromString("Hello world").Build()
	assert.NoError(t, err)

	hash, err := record.GetHash()
	assert.NoError(t, err)

	assert.Equal(t, "ed6c11b0b5b808960df26f5bfc471d04c1995b0ffd2055925ad1be28d6baadfd", hash)
}

File: languages/js/test/compat/compat.test.ts
import { describe, test, expect } from "@jest/globals";
import { RecordClient } from "../../dist";

describe("Compat Test", () => {
  test("compat test", async () => {
    let recordClient = new RecordClient();

    let record = await recordClient.fromString("Hello world").build();
    let hash = await record.getHash();

    expect(hash).toBe(
      "ed6c11b0b5b808960df26f5bfc471d04c1995b0ffd2055925ad1be28d6baadfd"
    );
  });
});
